include/mbedtls/chacha20.h

```diff
@@ -105,6 +105,32 @@ void mbedtls_chacha20_free(mbedtls_chacha20_context *ctx);
 int mbedtls_chacha20_setkey(mbedtls_chacha20_context *ctx,
                             const unsigned char key[32]);
 
+/**
+ * \brief           This function sets the variable length nonce and initial
+ *                  counter value.
+ *
+ * \note            A ChaCha20 context can be re-used with the same key by
+ *                  calling this function to change the nonce.
+ *
+ * \warning         You must never use the same nonce twice with the same key.
+ *                  This would void any confidentiality guarantees for the
+ *                  messages encrypted with the same nonce and key.
+ *
+ * \param ctx       The ChaCha20 context to which the nonce should be bound.
+ *                  It must be initialized and bound to a key.
+ * \param nonce     The nonce.
+ * \param nlen      The nonce length. This must be \c 12 or \c 8 Bytes in size.
```

Comment:
This is imprecise: `nelen` must have a value of 8 or 12, but it's not correct to say `nlen` must be 8 or 12 bytes in size (it's size is whatever size `size_t` is).